desktop/core/src/desktop/js/reactComponents/FileUploadQueue/FileUploadQueue.test.tsx

```diff
@@ -60,7 +60,7 @@ describe('FileUploadQueue', () => {
       <FileUploadQueue filesQueue={mockFilesQueue} onClose={() => {}} onComplete={() => {}} />
     );
 
-    const cancelButton = getAllByTestId('upload-queue__list__row__close-icon')[0];
+    const cancelButton = getAllByTestId('upload-queue__list__row__content__close-icon')[0];
```

Comment:
nit, suggestion call upload-queue__row-close-icon where the Block is "upload-queue" and the element is "row-close-icon"